apps/docs/package.json

```diff
@@ -18,6 +18,7 @@
         "autoprefixer": "^10.4.20",
         "daisyui": "^4.12.10",
         "postcss": "^8.4.41",
+        "sass": "^1.77.8",
```

Comment:
we dont need this

Reply:
resolved on https://github.com/coreproject-moe/monorepo/pull/126/commits/9d3b23352a88f364e214aea37f8c7df1b51966a8